### PR TITLE
fix missing help files for unidimensional reliability + some easy feature requests

### PR DESCRIPTION
**Pull request by juliuspfadt**
fixes https://github.com/jasp-stats/jasp-issues/issues/2175
will only work once a new `psych` version is on CRAN
fixes https://github.com/jasp-stats/jasp-issues/issues/1046
fixes https://github.com/jasp-stats/jasp-issues/issues/2655

### CODE REVIEW

File: R/raterAgreement.R [hunk 1]
```diff
@@ -48,12 +48,10 @@ raterAgreement <- function(jaspResults, dataset, options) {
...
   weighted <- options[["cohensKappaType"]] == "weighted"
 
-  weightedString <- ifelse(weighted, "Weighted", "Unweighted")
-
   # Create the JASP Table
-  jaspTable <- createJaspTable(title = gettextf("Cohen's %s kappa", weightedString))
+  jaspTable <- createJaspTable(title = gettext("Cohen's kappa"))
   jaspTable$addColumnInfo(name = "ratings", title = gettext("Ratings"), type = "string")
-  jaspTable$addColumnInfo(name = "cKappa", title = gettextf("%s kappa", weightedString), type = "number")
+  jaspTable$addColumnInfo(name = "cKappa", title = gettextf("kappa"), type = "number")
```
Comment (JTPetter) [inline R/raterAgreement.R:54]: You removed the dynamic naming, isn't it clearer when the output explicitly states if it is a weighted Kappa or not? Or maybe a footnote instead?

**Review by JTPetter — APPROVED** [review]

**Comment by JTPetter** [comment]
Otherwise no comments, I understand the unit tests should work once there is a new version of `psych`? My review does not count though, so you'll have to tag someone else :)

File: R/raterAgreement.R (hunk 1 above)
Comment (juliuspfadt) [reply R/raterAgreement.R:54]: Hmm, I thought about this, and it comes down to a design choice. I think that if a user clicks on weighted kappa they already expect to see the weighted kappa in the output, why add extra information that is already implied... But I suppose someone else might see this differently.

**Comment by juliuspfadt** [comment]
I suppose we @vandenman's approval to merge it.

**Review by boutinb — APPROVED** [review]

**Pull request merged**